modules/mcuboot/bl_storage.conf

```diff
@@ -0,0 +1,5 @@
+# This Kconfig fragment is applied to the MCUBoot configuration when
+# we want MCUBOOT to get access to the secure boot storage APIs
+# (bl_storage)
+CONFIG_SECURE_BOOT_STORAGE=y
+CONFIG_SECURE_BOOT_CRYPTO=y
```

Comment:
MCUBoot should not have to enable the SECURE_BOOT_STORAGE and SECURE_BOOT_CRYPTO libraries just to find out where the bl_monotonic_counters are stored.

We should encode the placement of the counters into partition manager instead.

That will reduce complexity and code size.